alpaca_trade_api/stream2.py

```diff
@@ -98,12 +100,44 @@ def __init__(self, key_id=None, secret_key=None, base_url=None):
             await self._ensure_nats()
             await self.polygon.subscribe(nats_channels)
 
+    async def test(self, poll_delay=1, max_buffer_length=1000, reconnect_delay=10, initial_channels=[]):
+        buffer = io.StringIO()
+        logger = logging.getLogger('websockets')
+        logger.setLevel(logging.DEBUG)
+        logger.addHandler(logging.StreamHandler(stream=buffer))
+
+        while True:
+            await asyncio.sleep(poll_delay)
+            buffer_text = buffer.getvalue()
+            if 'CLOSED' in buffer_text:
+                print('stream closed')
+                while True:
+                    try:
+                        print('attempting to reconnect...', end='\r')
```

Comment:
Is it possible to use the logger rather than prints for this kind of information? I would rather not print anything using print() - if we can't push it to the logger, I'd rather just not print anything and let exceptions bubble up when reconnection fails.

Reply:
I would drop the print statements altogether in production code; they are there only to demonstrate what is going on at this PR stage.